target/linux/ipq40xx/image/generic.mk

```diff
@@ -759,6 +759,19 @@ define Device/qxwlan_e2600ac-c2
 endef
 TARGET_DEVICES += qxwlan_e2600ac-c2
 
+define Device/telco_x1pro-lte
+	$(call Device/FitImage)
+	DEVICE_VENDOR := Telco
+	DEVICE_MODEL := X1 Pro
+	DEVICE_VARIANT := LTE
+	BOARD_NAME := x1pro-lte
```

Comment:
drop that.

Reply:
Drop these lines? Aren't they needed for menuconfig?

Reply:
BOARD_NAME should be dropped. Or do you need it for something?

Reply:
OK, I have dropped it